lib/charms/opensearch/v0/opensearch_secrets.py

```diff
@@ -69,18 +70,47 @@ def _on_secret_changed(self, event: SecretChangedEvent):
             logging.info(f"Label {event.secret.label} was meaningless for us, returning")
             return
 
+        system_user_hash_keys = [
+            self._charm.secrets.hash_key(user) for user in OpenSearchSystemUsers
+        ]
+        keys_to_process = system_user_hash_keys + [
+            CertType.APP_ADMIN.val,
+            self._charm.secrets.password_key(KibanaserverUser),
```

Comment:
I understand what you're doing (Kibanaserveruser needs to be updated in the client relation), but could we please add a comment here just so it's clear to future readers